cli/command_test.go

```diff
@@ -120,7 +119,7 @@ Usage: test COMMAND
 func TestRootCommand_Run(t *testing.T) {
 	t.Parallel()
 
-	ctx := logging.WithLogger(context.Background(), logging.TestLogger(t))
+	ctx := context.Background()
```

Comment:
Not really used and it causes circular dependency

Reply:
This raises a somewhat philosophical question of "should the logger belong to the CLI"... In my head, I was thinking of inverting the problem (making the CLI depend on the logger) so then it would be something like `f.LogLevelFlag(&logger)`.

Reply:
I started with something like that but then I thought it would be nice if the default logger could pick up the flags. Could be over stretching. Will just do the flags then.

Reply:
Tried a different approach. PTAL